view/frontend/web/js/view/catalog/badge.js

```diff
@@ -80,6 +92,30 @@ define([
         return price ;
     }
 
+    /**
+     *
+     * @param productId Id of product for Id concatenation
+     * @param subClass Sub class to select
+     * @returns {*}
+     */
+    function getHtmlPriceContainer(productId,subClass = false){
```

Comment:
can you add ` ` (space) after `,` (comma) please  😉

Reply:
Done